_posts/2020-03-23-one-click-install.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: "One-click install for Scala"
+---
+
+Installing Scala has always been a task more challenging than necessary, with the potential to drive away beginners.
+Should I install Scala itself? sbt? Some other build tools? What about a better REPL like Ammonite? Oh and before all that I need to install Java?
+
+To solve this problem, the Scala Center contracted Alexandre Archambault in January 2020 to add a one-click install of Scala to [coursier](https://get-coursier.io/docs/cli-overview).
```

Comment:
I’m not sure about “one-click install of Scala **to** coursier”. Should it be “through coursier” instead?